aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -32,6 +32,14 @@ Tensor dequantize_quant(const Tensor& self) {
   return get_qtensorimpl(self)->quantizer()->dequantize(self);
 }
 
+std::vector<Tensor> dequantize_tensors_quant(TensorList tensors) {
```

Comment:
Why is this needed? Can we handle lists in a more primitive manner? What if we need to add a dict of tensors, would we add one more function for this purpose?

In general, how are lists of tensors handled? How do we observe a list of tensors and quantize them?

Reply:
explained offline